browser-extension/mv2/src/background/background.js

```diff
@@ -1085,6 +1091,18 @@ BG.Methods.startRecordingOnUrl = (url) => {
   });
 };
 
+BG.Methods.showToast = (tabId, heading, subheading, iconPath) => {
+  const _showToast = `
```

Comment:
`_` prefix in variable name is used for private class members. It can be renamed to `showToastScript`.